browser/main/lib/ConfigManager.js

```diff
@@ -59,7 +59,10 @@ export const DEFAULT_CONFIG = {
     enableFrontMatterTitle: true,
     frontMatterTitleField: 'title',
     spellcheck: false,
-    enableSmartPaste: false
+    enableSmartPaste: false,
+    customMarkdownLintConfig: `{
```

Comment:
Please add a constant like following to the top of the file:
```js
const DEFAULT_MARKDOWN_LINT_CONFIG = `{
  "default": true
}`
```

And use it here like:
```suggestion
    customMarkdownLintConfig: DEFAULT_MARKDOWN_LINT_CONFIG 
```

The constant is used so you're not messing with the indentation. The indent added a weird looking output like in the following screenshot:
![grafik](https://user-images.githubusercontent.com/3046542/57402955-c8f6c180-71d8-11e9-9982-563819c1bbd3.png)

After the change it will look like:
![grafik](https://user-images.githubusercontent.com/3046542/57403075-170bc500-71d9-11e9-8aec-0391a1d569ec.png)